mono/metadata/mono-perfcounters.c

```diff
@@ -1387,7 +1394,8 @@ mono_perfcounter_update_value (void *impl, MonoBoolean do_incr, gint64 value)
 }
 
 void
-mono_perfcounter_free_data (void *impl)
+ves_icall_System_Diagnostics_PerformanceCounter_FreeData (void *impl, MonoError *error)
+// previously mono_perfcounter_free_data
```

Comment:
Why keeping that?

Reply:
It is much easier to sere here than in history and it has been useful looking at other branches at the same time.